clang/include/clang/Driver/Options.td

```diff
@@ -1822,6 +1822,9 @@ def fstrict_vtable_pointers: Flag<["-"], "fstrict-vtable-pointers">,
 def fstrict_overflow : Flag<["-"], "fstrict-overflow">, Group<f_Group>;
 def fintelfpga : Flag<["-"], "fintelfpga">, Group<f_Group>,
   Flags<[CC1Option, CoreOption]>, HelpText<"Perform ahead of time compilation for FPGA">;
+def fenable_usm_address_spaces : Flag<["-"], "fenable-usm-address-spaces">,
```

Comment:
Should this be `-fsycl-enable-...`? Myself, I don't really have an opinion here. On one hand, I gather that USM address spaces have become a common concept. On the other hand, the help text clearly brings the SPIR-V specifics, which kind of limits the option to DPC++.